tests/messages/properties/message_specific/test_bioscan_pool_xp_sample.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from tol_lab_share import error_codes
+from tol_lab_share.messages.properties.message_specific.bioscan_pool_xp_sample import BioscanPoolXpSample
+from tol_lab_share.messages.properties.simple.dict_value import Value
+
+
+@pytest.fixture
+def valid_sample():
+    return Value(
+        {
+            "sampleName": "ValidSample",
+            "sampleUuid": b"12345678-1234-1234-1234-1234567890ab",
+            "speciesName": "ValidSpecies",
+        }
+    )
+
+
+@pytest.fixture
+def invalid_sample():
+    return Value(
+        {
+            "sampleName": 42,
+            "sampleUuid": "12345678-1234-1234-1234-1234567890ab",
+        }
+    )
+
```

Comment:
Is this invalid because it misses speciesName only? Is there a problem with sampleUuid, i.e. bytes vs string.?